ReactiveCocoa/Swift/DynamicProperty.swift

```diff
@@ -66,3 +73,69 @@ import enum Result.NoError
 			}
 	}
 }
+
+extension DynamicProperty where Value: _ObjectiveCBridgeable {
+	/// Initializes a property that will observe and set the given key path of
+	/// the given object, where `Value` is a value type that is bridgeable
+	/// to Objective-C.
+	///
+	/// `object` must support weak references!
+	public convenience init(object: NSObject, keyPath: String) {
+		self.init(object: object, keyPath: keyPath, representation: BridgeableRepresentation())
+	}
+}
+
+extension DynamicProperty where Value: AnyObject {
+	/// Initializes a property that will observe and set the given key path of
+	/// the given object, where `Value` is a reference type that can be
+	/// represented directly in Objective-C via `AnyObject`.
+	///
+	/// `object` must support weak references!
+	public convenience init(object: NSObject, keyPath: String) {
+		self.init(object: object, keyPath: keyPath, representation: DirectRepresentation())
+	}
+}
+
+/// Represents values in Objective-C directly, via `AnyObject`.
+private struct DirectRepresentation<Value: AnyObject>: ObjectiveCRepresenting {
+	func extractValue(fromRepresentation representation: AnyObject) -> Value {
+		return representation as! Value
+	}
+
+	func represent(value value: Value) -> AnyObject {
+		return value
+	}
+}
+
+/// Represents values in Objective-C indirectly, via bridging.
+private struct BridgeableRepresentation<Value: _ObjectiveCBridgeable>: ObjectiveCRepresenting {
```

Comment:
Actually, this is the issue I discussed in the PR description. It's not possible to add protocol conformance in a protocol extension, so that's why I introduced a concrete type here.